content/blog/2022-02-28-using-iterative-terraform-provider-with-docker.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Using Iterative Terraform Provider with Docker
+date: 2022-02-28
+description: >
+  In this tutorial you'll learn how you can move use an existing Docker image with 
+  Iterative Terraform Provider, as well as build your own Docker image.
+descriptionLong: >
+  In this tutorial you'll learn how you can move use an existing Docker image with 
+  Iterative Terraform Provider, as well as build your own Docker image.
+picture: 2022-02-28/unsplash-containers.jpg
+author: maria_khalusova
```

Comment:
The branch can be deployed successfully once we [add a author to `content/authors`](https://dvc.org/doc/user-guide/contributing/blog#adding-authors).

Reply:
Right, the author is added in a different PR which will be published before this one, so I'm not even sure if it makes sense to add here as well :D

Reply:
It makes some sense, so we can see the review app.

What's the other PR? Maybe that one is ready to merge

Reply:
Nope, it's not. It's the other tutorial that will be published after the release of TPI.
So here's the sequence. First, there will be a release blog post (currently in google docs), then the blog post with the tutorial #1, then this one. I'm building up a backlog of posts.

Reply:
Not sure why github decided to make that a link to an issue, please ignore that link.

Reply:
OK np. Please request my review here once https://github.com/iterative/dvc.org/pull/3087 is merged (and `master` merged back into this PR's branch 🙂). Thanks